### ISSUE TITLE
positions of a particular contract?

### ISSUE THREAD

**Issue by dccw** [erdewit/ib_insync#51]
hello
ib.positions() lists all contracts under the account. I was wondering if there is any way to get the position of a particular contract. Thanks.

**Comment by erdewit** [comment]
It's easy to filter out, like

```python
contract = ...
position = next((p for p in ib.positions() if p.contract is contract), None)
```

**Comment by dccw** [comment]
maybe this is a separate issue but does ib.positions() track the positions real time? I m running on a simulated account and positions() does not seem to reflect the new positions arising from new orders. the new positions are reflected only after i close the current spyder instance and open a new one to run ib.positions() again.

**Comment by erdewit** [comment]
For spyder (and Jupyter notebooks) you'll have to call util.startLoop() to start an event loop that keeps things updated in realtime.
